buildpack/telemetry/fluentbit.py

```diff
@@ -139,9 +177,18 @@ def is_fluentbit_enabled():
     """
     The function checks if some modules which requires
     Fluent Bit is configured.
-
     """
 
     return any(
-        [splunk.is_splunk_enabled()]
+        [splunk.is_splunk_enabled(), newrelic.is_enabled()]
     )  # Add other modules, where Fluent Bit is used
+
+
+def _get_log_file() -> str:
+    """Discard logs unless debug is active."""
+    # FluentBit currently does not support log rotation,
+    # so the file must only be used when debugging
+    fluentbit_debug = os.getenv("FLUENTBIT_DEBUG", "false").lower()
```

Comment:
Distutils is deprecated and going to be removed in Python 3.12, so I think it's better to avoid.

However, we do have a `strtobool` method on `lib.m2ee.utils`, so I can use that.